### PR TITLE
dev-util/kbuild: add new revision 3572

### PR DESCRIPTION
**Pull request by ceamac**
Fix some long-standing bugs.  I didn't touch the stable version but applied them to the others.

### CODE REVIEW

File: dev-util/kbuild/files/kbuild-0.1.9998.3572-fix-bison.patch [hunk 1]
```diff
@@ -0,0 +1,15 @@
+--- a/src/kash/Makefile.kmk	2022-11-12 16:42:14.195927909 +0200
++++ b/src/kash/Makefile.kmk	2022-11-12 17:09:46.212159615 +0200
+@@ -210,8 +210,11 @@
+ kash_USES = lex yacc
+ kash_LEXTOOL = FLEX
+ kash_LEXFLAGS = -8
+-#kash_YACCTOOL = BISON
++ifneq (,$(shell which bison))
```
Comment (thesamesam) [inline dev-util/kbuild/files/kbuild-0.1.9998.3572-fix-bison.patch:8]: which is non-portable and not guaranteed by POSIX, we want to remove it from the system set in Gentoo eventually

Comment (ceamac) [reply dev-util/kbuild/files/kbuild-0.1.9998.3572-fix-bison.patch:8]: I can replace it by `command -v`

Comment (thesamesam) [reply dev-util/kbuild/files/kbuild-0.1.9998.3572-fix-bison.patch:8]: Sounds good!

Comment (ceamac) [reply dev-util/kbuild/files/kbuild-0.1.9998.3572-fix-bison.patch]: Doesn't work with `command -v`.  But I can try to run `bison -V`, if it works use it, if not use yacc.

**Pull request closed without merging**